packages/statemanager/test/statelessVerkleStateManager.spec.ts

```diff
@@ -1,9 +1,18 @@
 import { Block } from '@ethereumjs/block'
 import { Common } from '@ethereumjs/common'
-import { Address, bytesToHex } from '@ethereumjs/util'
+import {
+  Account,
+  Address,
+  bytesToBigInt,
+  bytesToHex,
+  hexToBytes,
+  randomBytes,
+} from '@ethereumjs/util'
+import { getStem } from '@ethereumjs/verkle'
+import { skip } from 'tape'
```

Comment:
Can we remove the `tape` usage?

Reply:
Can be replaced by the vitest equivalent: https://vitest.dev/api/#test-skip

Reply:
fixed